### PR TITLE
Finish updating guides: production

### PR DESCRIPTION
**Pull request by EugeneHlushko**
* Finish updating production guide to v4
* Small auto formatting in package.json

### CODE REVIEW

File: src/content/guides/production.md [hunk 1]
```diff
@@ -234,6 +192,37 @@ __src/index.js__
 ```
 
 
+## Minification
```
Comment (montogeek) [inline src/content/guides/production.md]: Why this structure change? What did you changed exactly?

Comment (EugeneHlushko) [reply src/content/guides/production.md]: for minification specifically or?
If yes:
-Add: webpack v4+ will minify your code by default in [`production mode`](/concepts/mode/#mode-production).
-Change: If you decide to try another minification plugin, just make sure your new choice also drops dead code as described in the [tree shaking](/guides/tree-shaking) guide and provide it as the [`optimization.minimizer`](/configuration/optimization/#optimization-minimizer).

File: src/content/guides/production.md [hunk 2]
```diff
@@ -148,42 +148,6 @@ __package.json__
...
-+       sourceMap: true
-+     })
-    ]
-  });
-```
-
-T> Avoid `inline-***` and `eval-***` use in production as they can increase bundle size and reduce the overall performance.
-
-
 ## Specify the Mode
```
Comment (EugeneHlushko) [inline src/content/guides/production.md]: Mode is now higher, as it is set key to both next paragraphs, and was coming after them in prev structure.

File: src/content/guides/production.md [hunk 3]
```diff
@@ -192,17 +156,11 @@ __webpack.prod.js__
 
 ``` diff
   const merge = require('webpack-merge');
-  const UglifyJSPlugin = require('uglifyjs-webpack-plugin');
```
Comment (EugeneHlushko) [inline src/content/guides/production.md]: this is webpack v3

File: src/content/guides/production.md [hunk 4]
```diff
@@ -192,17 +156,11 @@ __webpack.prod.js__
...
   const common = require('./webpack.common.js');
 
   module.exports = merge(common, {
     mode: 'production',
-    devtool: 'source-map',
```
Comment (EugeneHlushko) [inline src/content/guides/production.md]: this is webpack v3

File: src/content/guides/production.md [hunk 5]
```diff
@@ -148,42 +148,6 @@ __package.json__
...
-
-``` diff
-  const merge = require('webpack-merge');
-  const UglifyJSPlugin = require('uglifyjs-webpack-plugin');
-  const common = require('./webpack.common.js');
-
-  module.exports = merge(common, {
-    mode: 'production',
-+   devtool: 'source-map',
-    plugins: [
```
Comment (EugeneHlushko) [inline src/content/guides/production.md]: this was removed, webpack v3

**Comment by EugeneHlushko** [comment]
Please tell me if there are any more things unclear, or best run this version locally and compare with production version, it makes a lot of sense

File: src/content/guides/production.md (hunk 2 above)
Comment (montogeek) [reply src/content/guides/production.md]: I see, thanks

File: src/content/guides/production.md (hunk 1 above)
Comment (montogeek) [reply src/content/guides/production.md]: I mean, moving those 2 sections from the top to the bottom.

**Comment by montogeek** [comment]
Thanks!

**Pull request merged**